### DISCUSSION TITLE
Trees aren't rendering past a certain distance.

### DISCUSSION THREAD

**Discussion by Wolfos** [visualdesigncafe/nature-renderer#22] [Help]
I can't seem to increase the tree render distance past 400-something units. Any value past that won't actually increase the distance, no matter where I set it.

It's not the LOD distance - they render just fine if I let Unity do it. It's not the render distance on the trees themselves, which makes no difference when increased past the limit, nor is it the default settings. What else could be overriding the maximum?

Using HDRP and Nature Renderer 2022

**Comment by maxplooi** [comment]
Other settings you could check are the [Camera Settings](https://visualdesigncafe.com/nature-renderer/docs/2022/camera-settings) that may be limiting the render distance, or the "Density in Distance" setting that may be reducing the density to 0 (nothing).

If changing those doesn't work then could you share some screenshots of your prefab, terrain, and settings? Then I'll check them for you.

**Reply by Wolfos** [reply]
I posted those below. The only value that has any effect on this render distance is the LOD bias in Nature Renderer camera settings. However changing the culling distance in the prefab's LOD group has no effect whatsoever. It also doesn't match the actual prefab culling distance, suggesting this is being miscalculated.

Ideally the culling distance in LOD group is ignored entirely, with nature renderer handling it instead, as LOD and drawing distance should be configured separately.

Bumping up the LOD bias is not an acceptable workaround as it affects both culling distance and the LOD transition.

**Comment by Wolfos** [comment]
## Prefab
No overrides here
<img width="570" alt="Screenshot 2024-03-07 at 17 57 31" src="https://github.com/visualdesigncafe/nature-renderer/assets/695652/69d926d5-6e0d-4fa4-abe5-56e42f39f44d">

## Terrain
<img width="377" alt="Screenshot 2024-03-07 at 18 15 36" src="https://github.com/visualdesigncafe/nature-renderer/assets/695652/49b5005d-8749-4df8-889f-00ec0f6e7a86">


## Camera
<img width="383" alt="Screenshot 2024-03-07 at 18 04 31" src="https://github.com/visualdesigncafe/nature-renderer/assets/695652/2be7bbc2-53b8-4e40-b055-5b0042f02d58">

## Global settings
Changing density / range here doesn't matter, as the trees stop rendering long before the 5000 unit mark
<img width="392" alt="Screenshot 2024-03-07 at 18 06 31" src="https://github.com/visualdesigncafe/nature-renderer/assets/695652/1d917c1e-6edc-4224-aa3a-3f1a4daa1b5d">

## Trees rendered by Unity
<img width="889" alt="Screenshot 2024-03-07 at 18 08 26" src="https://github.com/visualdesigncafe/nature-renderer/assets/695652/256d433e-001d-4070-8467-d827f71a870a">

## Trees rendered by Nature Renderer
<img width="889" alt="Screenshot 2024-03-07 at 18 08 48" src="https://github.com/visualdesigncafe/nature-renderer/assets/695652/fb52c205-f9af-48fb-9015-23a99c6fcab3">

**Comment by maxplooi** [comment]
It looks like this may be caused due to a compatibility issue with Amplify Imposters. Could you try using a prefab that does not use Amplify Imposters to check if that works correctly? I believe Amplify requires a specific rendering setup for their imposters, although I don't own Amplify Imposters myself so I can't easily verify it for you.

**Reply by Wolfos** [reply]
Tested by replacing the impostor LOD - it does render further now, but there's still a limit.
